types.go

```diff
@@ -237,14 +237,12 @@ type Key struct {
 
 // This structure is used to send the exact ISO-8601 time format managed by Meilisearch
 type KeyParsed struct {
-	Name        string    `json:"name"`
-	Description string    `json:"description"`
-	UID         string    `json:"uid,omitempty"`
-	Actions     []string  `json:"actions,omitempty"`
-	Indexes     []string  `json:"indexes,omitempty"`
-	CreatedAt   time.Time `json:"createdAt,omitempty"`
-	UpdatedAt   time.Time `json:"updatedAt,omitempty"`
-	ExpiresAt   *string   `json:"expiresAt"`
+	Name        string   `json:"name"`
+	Description string   `json:"description"`
+	UID         string   `json:"uid,omitempty"`
+	Actions     []string `json:"actions,omitempty"`
+	Indexes     []string `json:"indexes,omitempty"`
+	ExpiresAt   *string  `json:"expiresAt"`
```

Comment:
Sorry @alallema but I don't remember why you had to remove this, can you add the reason in the PR description or somewhere else?

Reply:
Also, this is a breaking, so you need to use `breaking-change` right?

Reply:
I'm not sure because yes, it breaks because the `struct KeyParsed` is modified but normally the user doesn't have to use this structure. Its use is limited internally to the `CreateKey` method.

Reply:
If the user can touch it, and it change it is breaking :D This is the case?